evidence/pool.go

```diff
@@ -306,6 +305,16 @@ func (evpool *Pool) SetLogger(l log.Logger) {
 	evpool.logger = l
 }
 
+// GetHeaderAtHeight gets the header from the block store at a specified height.
+// Is used for validation of LunaticValidatorEvidence
+func (evpool *Pool) GetHeaderAtHeight(height int64) *types.Header {
+	blockMeta := evpool.blockStore.LoadBlockMeta(height)
```

Comment:
do you think we should just add `blockStore` to state?

Reply:
I thought about it but I'm generally against it because blockStore is about the history and state is the current state so I don't think it's a good idea semantically to mix them